.storybook/preview.tsx

```diff
@@ -1,7 +1,7 @@
 import { Preview } from '@storybook/react';
-import { Box } from '../packages/react/src/box';
-import { Core } from '../packages/react/src/core';
-import { theme as agriculture } from '../packages/react/src/ag-branding';
+import { Box } from '@ag.ds-next/react/src/box';
```

Comment:
What prompted this change? Should the stories be updated with package paths now, too?

Reply:
Ah just an auto import, I'll revert.

- [ ] Box import relative path